src/Fantomas/paket.references

```diff
@@ -1,4 +1,6 @@
 FSharp.Compiler.Service
 FSharp.Core
 Ionide.KeepAChangelog.Tasks copy_local: true
-Dotnet.ReproducibleBuilds copy_local: true
\ No newline at end of file
+Dotnet.ReproducibleBuilds copy_local: true
+System.IO.Abstractions
```

Comment:
Are these references required in `Fantomas`?
I would assume you need them in `Fantomas.Extras` instead.

Reply:
![image](https://user-images.githubusercontent.com/2621499/160278000-bde3ce6e-bdcd-4868-a1fd-23cc4a992b69.png)

This introduces unnecessary dependencies in `Fantomas`.

`System.IO.Abstractions` should be used in `Fantomas.Extras` and `System.IO.Abstractions.Tests` in `Fantomas.Tests` I think.

Reply:
Oh, this is simple accident - just me not being used to Paket, sorry.